File: ProjectPal/Dtos/StudySessionView.cs
namespace ProjectPal.Dtos;

public class StudySessionView
{
    public int StudySessionId { get; set; }
    public string Topic { get; set; }
    public IEnumerable<StudySessionLink> Links { get; set; }
    public IEnumerable<StudySessionTag> Tags { get; set; }
    public int MinutesStudied { get; set; }
    public string Notes { get; set; }
    public DateTimeOffset DateStudied { get; set; }
}

File: ProjectPal/Client/src/app/Dtos/studySessionView.ts
import { StudySessionLink } from "./studySessionLink";
import { StudySessionTag } from "./studySessionTag";

export interface StudySessionView {
    studySessionId: number;
    topic: string;
    links: StudySessionLink[];
    tags: StudySessionTag[];
    minutesStudied: number;
    notes: string;
    dateStudied: string;
}